watttime/api.py

```diff
@@ -70,6 +70,19 @@ def _get_chunks(
         # API response is inclusive, avoid overlap in chunks
         chunks = [(s, e - timedelta(minutes=5)) for s, e in chunks[0:-1]] + [chunks[-1]]
         return chunks
+    
+    def register(self, email: str, organization: Optional[str] = None) -> requests.Response:
+        """Register for the WattTime API, if you do not already have an account."""
+        url = f"{self.url_base}/register"
+        params = {
+            'username': self.username,
+            'password': self.password,
+            'email': email,
+            'org': organization,
+        }
+             
+        rsp = requests.post(url, json=params, timeout=20)
+        return rsp
```

Comment:
How do you feel about changing the return type to a `bool` of whether it was successful and raising an error on non-200 response code? I think not everyone will be familiar with response object or that they should check that it's successful.

Reply:
good with me! I also want to add a `rsp.raise_for_status()` within api.py after each call, as it may make debugging easier for users (e.g. 429s). I was going to do that in a separate PR.

Reply:
sounds good, this function might not even need a return if it is only either a) successful or b) raises and error

Reply:
changed, to not return anything, but rather to print a message to check your email if no errors were raised. 

In the future, we should implement proper logging throughout